qt/scientific_interfaces/Inelastic/BayesFitting/Quasi.cpp

```diff
@@ -518,13 +528,16 @@ void Quasi::plotClicked() {
   auto const resultWS = AnalysisDataService::Instance().retrieveWS<MatrixWorkspace>(resultName);
   int const numSpectra = (int)resultWS->getNumberHistograms();
   IndirectTab::checkADSForPlotSaveWorkspace(resultName, true);
-  auto const paramNames = {"Amplitude", "FWHM", "Beta"};
+  auto const paramNames = {"amplitude", "fwhm", "beta"};
   for (std::string const &paramName : paramNames) {
 
-    if (plot == paramName || plot == "All") {
+    if (plot == paramName || plot == "all") {
       std::vector<std::size_t> spectraIndices = {};
       for (auto i = 0u; i < static_cast<std::size_t>(numSpectra); i++) {
         auto axisLabel = resultWS->getAxis(1)->label(i);
+        // Convert to lower case
+        std::transform(axisLabel.begin(), axisLabel.end(), axisLabel.begin(),
+                       [](unsigned char c) { return std::tolower(c); });
```

Comment:
Just curious, why you want these labels in lower case?

Reply:
There are a few inconsistencies between the BayesQuasi2 (quickBayes) and BayesQuasi (quasielasticbayes) algorithms. BayesQuasi2 produces a result workspace with y axis labels which have the first letter capitalised, whereas BayesQuasi does not. This makes the Quasi interface work with both algorithms by not caring about the capitalization of the y axis labels.

Reply:
Makes sense, thank you.